pkg/project/app_classic/Taskfile.yml

```diff
@@ -252,6 +251,7 @@ tasks:
   dispatcher:destroy:
     desc: destroy AEM dispatcher
     dir: dispatcher
+    deps: [ dispatcher:stop ]
```

Comment:
I am not sure if this rly needed...

Reply:
I assumed it was, as `dispatcher:stop` was already being called indirectly when running `stop` task. Tomorrow I will check if this is needed.